### DISCUSSION TITLE
bug: Extraneous kwargs provided to the energy storage package

### DISCUSSION THREAD

**Discussion by brixel-1984** [modflowpy/flopy#2251] [Q&A] [answered]
I saw that a series of notebooks have been published to illustrate the capacities of the new Groundwater Energy (GWE) model of modflow 6.

When trying to run these notebooks, I get the following error (using either flopy versions 3.7.0 of 3.8.0dev):
`FlopyException: Extraneous kwargs "heat_capacity_water, density_water" provided to MFPackage. ()`

The issue seems to be with the two arguments ('heat_capacity_water' and 'density_water') provided to the energy storage package, as they do not show up in the docstring.

```
flopy.mf6.ModflowGweest(
    model,
    loading_package=False,
    save_flows=None,
    zero_order_decay=None,
    latent_heat_vaporization=None,
    porosity=None,
    decay=None,
    cps=None,
    rhos=None,
    packagedata=None,
    filename=None,
    pname=None,
    **kwargs,
)
Docstring:     
ModflowGweest defines a est package within a gwe6 model.
```

And taking them out creates then a problem with the .nam file...

Any clue?

**Comment by christianlangevin** [comment, marked as answer]
Hi @brixel-1984, sorry for this inconvenience.  We changed the input slightly for the GWE-EST Package in the both the "develop" version of MODFLOW 6 and the develop version of FloPy.  So for these notebooks to work, you will need the last release of both MODFLOW and FloPy or the develop versions of both MODFLOW 6 and FloPy.

You can get develop versions of MODFLOW 6 [here](https://github.com/MODFLOW-USGS/modflow6-nightly-build/releases).

**Comment by wpbonelli** [comment]
@langevin-usgs please correct me if I'm wrong but @brixel-1984 if you opt for the develop versions, I believe you will also need to [regenerate FloPy MF6 package classes](https://flopy.readthedocs.io/en/latest/md/generate_classes.html#generating-flopy-classes), as the updated inputs have not yet been checked into the FloPy repository (typically we do this just before release time).

**Comment by christianlangevin** [comment]
Yes, good point, @wpbonelli.
